File: easy-solutions/cpp-solutions/palindrome-number.cpp
#include <iostream>
class Solution {
public:
    bool isPalindrome(int x) {
        int reverse=x%10;
        int original = x;
        if(x>0){
            while(x/10>0 ){
                //std::cout << "x " << x << std::endl;
                //std::cout << "reverse " << reverse << std::endl;
                x=x/10;
                reverse=reverse*10 + x%10;
            }
        }
        if(original==reverse){
            return true;
        }
        return false;
    }
};

File: easy-solutions/python-solutions/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        reverse = x%10
        original = x
        if(x>0):
            while(x/10>0):
                x=x/10
                reverse=reverse*10 + x%10
        
        if(reverse == original):
            return True
        
        return False
            

        